Sources/Controllers/2.0/OAPOIViewController.mm

```diff
@@ -187,7 +187,8 @@ - (BOOL) showNearestWiki
 - (void) buildRows:(NSMutableArray<OARowInfo *> *)rows
 {
     NSString *prefLang = [OAUtilities preferredLang];
-    
+    NSArray <NSString *> *clickableContactCells = @[@"youtube", @"facebook", @"instagram", @"twitter", @"vk", @"ok", @"webcam", @"telegram", @"linkedin", @"pinterest", @"foursquare", @"xing", @"flickr", @"email", @"mastodon", @"diaspora", @"gnusocial", @"skype"];
+    NSArray *phoneNumberCells = @[@"phone", @"mobile", @"whatsapp", @"viber"];
```

Comment:
Line 191:```NSArray <NSString *> *```
Line 192:```NSArray *``` ???
Secondly, if a collection is used for ```containsObject``` operation only, it is always preferable to use NSSet.
Thirdly, extract constant collections to static private variables